src/app/utils/tests/MGPValidation.spec.ts

```diff
@@ -2,6 +2,18 @@
 import { MGPFallible } from '../MGPFallible';
 import { MGPValidation } from '../MGPValidation';
 
+export class MGPValidationTestUtils {
+
+    public static expectToBeSuccess(fallible: MGPValidation): void {
+        expect(fallible.isSuccess()).toBeTrue();
+    }
+
+    public static expectToBeFailure(fallible: MGPValidation, reason: string): void {
+        expect(fallible.isFailure()).toBeTrue();
+        expect(fallible.getReason()).toBe(reason);
+    }
+}
+
```

Comment:
do we need it, bu qu'on a ça pour les MGPFallible<pas void> déjà ?

Reply:
pour traiter le cas des `MGPValidation` c'est plus pratique, car dans les fallibles il attend une valeur contenue, de type `void`, donc faudrait mettre du undefined ou autre, ce qui serait pas topitop (et fonctionnait pas en plus je crois)